design/cryptography.md

```diff
@@ -103,7 +118,6 @@ An example would be:
 - Validators do not reveal keys so that keys are not susceptible to attacks
```

Comment:
this is mixing requirements with proposed solutions.

Validators not revealing is a proposed solution to which problem? What are the tradeoffs? 
What is the alternative?

Reply:
I've reworded it. This feature/requirement is what we discussed a few days ago.

The way I see it, it's good that the Validators release the keys ( load-spreading) but there's no advantage for the Validators to calculate them.
In other words, the Validators can be queried by a key that's been released by the Central Sequencer, but they will never calculate and release them.

WDYT ?

Reply:
I'm not passing judgement about that solution. It might be an acceptable phase 1 or not.
What I'm saying is that for a reader of this document it should be clear what problem is being solved. Then a couple of proposed solutions each with advantages/tradeoffs

Reply:
Got it. Following the format of the document I can write that in the form of a question + answer. I feel like a more in-depth evaluation probably requires a design of its own given the amplitude of possibilities. 

Maybe for the first phase, this is good enough and I can frame that in ?